src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,23 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private double armLength;
```

Comment:
Don't forget about access modifiers for your custom fields in `Excavator` class. It's a good practice to make fields private and provide access through getters and setters.